.github/workflows/3-staging-to-prod.yml

```diff
@@ -1,63 +1,25 @@
-name: 3 | Generate PR for releases/staging to releases/prod
+name: 3 | Push releases/staging to releases/prod
 
-# This CI job is responsible for generating PRs that bring the HEAD of `releases/staging` into `releases/prod`.
-# These PRs are meant to be the only (standard) way that code is merged into the `releases/prod` branch.
+# This CI job is responsible for force pushing the content of releases/staging to releases/prod. It
+# is restricted to web-reviewers through virtue of the GitHub environment protection rules for the
+# prod environment.
 
 on:
   workflow_dispatch:
 
-# https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#keep-a-branch-up-to-date-with-another
 jobs:
-  prod-gen-pr:
-    name: 'Generate PR for merging to releases/prod branch'
+  push-prod:
+    name: 'Push to prod branch'
     runs-on: ubuntu-latest
     environment:
-      name: release
+      name: prod
```

Comment:
Continue to use `release` for now - the service account is only set up in this environment.

Reply:
use `push/prod` here, actually

Reply:
Done!